package.json

```
@@ -19,24 +19,24 @@
     ]
   },
   "dependencies": {
-    "@hapi/b64": "5.x.x",
-    "@hapi/boom": "9.x.x",
-    "@hapi/bounce": "2.x.x",
-    "@hapi/bourne": "2.x.x",
-    "@hapi/catbox-object": "2.x.x",
-    "@hapi/cryptiles": "5.x.x",
-    "@hapi/hoek": "9.x.x",
-    "@hapi/wreck": "17.x.x",
-    "ecdsa-sig-formatter": "1.x.x",
+    "@hapi/b64": "^6.0.0",
+    "@hapi/boom": "^10.0.0",
+    "@hapi/bounce": "^3.0.0",
+    "@hapi/bourne": "^3.0.0",
+    "@hapi/catbox-object": "^2.0.0",
```

One outstanding TODO before marking this PR as not a draft.
```suggestion
    "@hapi/catbox-object": "^3.0.0",
```